src/platform/forms-system/src/js/utilities/save-in-progress-messages.js

```diff
@@ -0,0 +1,19 @@
+import { APP_TYPE_DEFAULT } from '../constants';
+
+const appType = formConfig =>
+  formConfig.customText?.appType || APP_TYPE_DEFAULT;
+
+export function inProgressMessage(formConfig) {
+  const defaultMessage = `Your ${appType} is in progress.`;
+  return formConfig.saveInProgress?.inProgress || defaultMessage;
+}
+
+export function expiredMessage(formConfig) {
+  const defaultMessage = `Your saved ${appType} has expired. If you want to apply for benefits, please start a new ${appType}.`;
```

Comment:
So, this means that we're treating the first occurrence of `appType` as the old `formDescriptions[formId]`, and hardcoding the "description" to be `benefits`. These seem like sensible defaults.